src/tools/rbd/Utils.cc

```diff
@@ -1078,5 +1078,74 @@ bool check_if_image_spec_present(const po::variables_map &vm,
   return false;
 }
 
+int write_zero_data(librbd::Image &image)
```

Comment:
Nit: since this only applies to ```create```, you can put the code in ```Create.cc``` since it isn't going to be used by other actions.